### ISSUE TITLE
adding a simple GUI to the vrecord interface

### ISSUE THREAD

**Issue by tinytomato** [amiaopensource/vrecord#629]
@dericed and I have been discussing adding a simple GUI form for vrecord that would be more accessible to users than the form as it is now (image below, but you all know it well!). This simple GUI would include: 

- a summary of the settings that when clicked prompts the user back to the edit settings page

- minimum viable entry information 

What are the necessary elements of the vrecord GUI that are input for every capture session? What elements are not changed 90% of the time and only need to be reset for specific instances?

Basically this Simple GUI is an attempt to turn this
<img width="1322" alt="Screen Shot 2020-10-26 at 4 53 21 PM" src="https://user-images.githubusercontent.com/56849931/97229002-17719480-17ae-11eb-88df-b2b9c3aba834.png">
into this
<img width="1316" alt="Screen Shot 2020-10-26 at 4 59 03 PM" src="https://user-images.githubusercontent.com/56849931/97229019-1b051b80-17ae-11eb-850e-cf554af5e59d.png">

Let's discuss what this Simple GUI could look like and make vrecord a little easier to operate on the daily.

**Comment by iamdamosuzuki** [comment]
I think this is a really great idea. We could design one or two simple menus based off use cases. Would anyone be interested in a zoom call to discuss this and maybe do some prototyping? How about the first week of November? Or we could do it during an AMIA night?

@ablwr @privatezero @kieranjol @dericed @bturkus @tinytomato @retokromer @libbyhopfauf @aeschweik (sorry if i missed anyone in the tags!)

**Comment by privatezero** [comment]
I do like the idea of multiple menus, for example having it fire up with a unique stripped down menu for whatever recording mode it is set to! Also could possibly be integrated with options currently available via `vrecord -g`

**Comment by BleakleyMcD** [comment]
I think is a good idea. 

Not to push this off topic, but in my experience, and especially using vrecord with people who aren’t very familiar with it, the GUI is not the part which causes the most confusion. It has stuff most people understand or can interpret. The bigger confusion that causes people to question whether to use it at all is the ffmpeg output after the recording is done. I think few people, myself included, can really interpret that well and output like buffer overrun repeated 152 time or dropped frames or other odd ffmpeg syntax flags is very confusing to know if your file is good even if it passes the mediaconch integrity check. Sure this is where post-digitization QC comes in, but I’d be very interested in working on something that helps explain in detail what all that ffmpeg output is and what to worry about and what not to. 

Down to help on the GUI too!

**Comment by XFRCollective** [comment]
Love this idea, would love to have more of a boxy rectangular shape and responsive window like 
vrecord -g. Also maybe a checkbox that asks if your logs are in the same directory as the digital files so you don't have go searching a second time for the same directory. 

+1 what @BleakleyMcD said. We tend to just ignore the reports and and use our eyes/ears/mediainfo in the end.

**Comment by tinytomato** [comment]
Thanks @XFRCollective @BleakleyMcD @iamdamosuzuki @privatezero for the feedback/interest!

@XFRCollective --> Definitely aiming for boxy and responsive, with the simple GUI becoming an option in vrecord -g. Also ty for your suggestion for checking if the logs are in the same directory. 

It would be great to have feedback on use cases. I'm starting work on a mock up of the form and very down to discuss during AMIA next week!

**Comment by retokromer** [comment]
Sorry for my late answer. In my opinion too, it would make things clearer for the users.

**Comment by libbyhopfauf** [comment]
Like others have said, I think this would make it more efficient vs. having to go in and go through all of the settings every time. It would also provide a sort of QC on settings being changed. And I agree with @BleakleyMcD that there needs to be some more documentation on the errors/warning/information being generated in the Terminal to make it easy for people to know what they can ignore and what they need to worry about. Sort of like the table with error codes for video decks 🤓🧐

**Comment by tinytomato** [comment]
Hi everyone - I have a pull request up for review and would appreciate comments and feedback on the design as well as some other questions I ran into while making it. ty!

https://github.com/amiaopensource/vrecord/pull/644

**Comment by dericed** [comment]
Closed by https://github.com/amiaopensource/vrecord/pull/653 :) Thx @tinytomato!
